maven-core/src/main/java/org/apache/maven/execution/BuildResumptionAnalyzer.java

```diff
@@ -32,5 +32,5 @@
      * @param result Outcome of the current Maven build.
      * @return A {@link BuildResumptionData} instance or {@link Optional#empty()} if resuming the build is not possible.
      */
-    Optional<BuildResumptionData> determineBuildResumptionData( final MavenExecutionResult result );
+    Optional<BuildResumptionData> determineBuildResumptionData( MavenExecutionResult result );
```

Comment:
Why did you remove the `final` keywords?

Reply:
They break recent checkstyle rules, which makes sense because they have no meaning on method declaration afaik.